dir.props

```diff
@@ -52,6 +52,7 @@
       https://www.myget.org/F/nugetbuild/api/v3/index.json;
       https://api.nuget.org/v3/index.json;
       https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json;
+      $(RestoreSources);
```

Comment:
Should this be prepended, as per the title, rather than appended?

Reply:
`RestoreSources` is being prepended *to*. Regardless of wording, this is how CoreFX does it: https://github.com/dotnet/corefx/blob/ea8fe5d29798dbfa2a1ed33341b788b61aa59795/dir.props#L80-L81.